### ISSUE TITLE
Load project hangs due to EACCES: permission denied

### ISSUE THREAD

**Issue by willhoyle** [vuejs/vetur#2559]
- [x] I have searched through existing issues
- [x] I have read through [docs](https://vuejs.github.io/vetur)
- [x] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)
- [x] I have tried restarting VS Code or running `Vetur: Restart VLS`

## Info

- Platform: Linux
- Vetur version: v0.31.2  v0.31.1  v0.31.0 (works when using v0.30.3)
- VS Code version: 1.52.0

## Problem

Vetur initialized
[Error - 11:45:17 AM] Request textDocument/documentSymbol failed.
  Message: Request textDocument/documentSymbol failed with message: EACCES: permission denied, scandir '/home/will/projects/dashboard/data/postgres'
  Code: -32603 

## Reproducible Case

I have a workspace with a repo similar to the following:

```
dashboard/
    data/postgres
    frontend/
        components/
        pages/
        package.json
        etc...
    src/backend stuff/...
    vetur.config.js
```

since my vue app is in frontend, I have the following vetur.config.js (ie. dashboard/vetur.config.js)

```js
// vetur.config.js
/** @type {import('vls').VeturConfig} */
module.exports = {
    // **optional** default: `{}`
    // override vscode settings
    // Notice: It only affects the settings used by Vetur.
    settings: {
      "vetur.useWorkspaceDependencies": true,
      "vetur.experimental.templateInterpolationService": true
    },
    // **optional** default: `[{ root: './' }]`
    // support monorepos
    projects: [
      {
        // **required**
        // Where is your project?
        // It is relative to `vetur.config.js`.
        root: './frontend',
        // **optional** default: `'package.json'`
        // Where is `package.json` in the project?
        // We use it to determine the version of vue.
        // It is relative to root property.
        package: './package.json',
        // **optional** default: `[]`
        // Register globally Vue component glob.
        // If you set it, you can get completion by that components.
        // It is relative to root property.
        // Notice: It won't actually do it. You need to use `require.context` or `Vue.component`
      }
    ]
  }
```
my `.gitignore` has `data` in it. It is managed by the postgres docker image, which is why I don't have permission to read   the folder. I could fix this by fixing how I run that docker image but it would be nice to have vetur ignore this folder since it doesn't have permission? 

Also, I'll just link the other issue for reference: https://github.com/vuejs/vetur/issues/2554

**Comment by yoyo930021** [comment]
[vetur-0.31.2.vsix.zip](https://github.com/vuejs/vetur/files/5682562/vetur-0.31.2.vsix.zip)

https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix

Install this version and paste output panel message in this issue.

**Comment by yoyo930021** [comment]
I can't reproduce in my macOS.
Please provide a repro case.
https://github.com/vuejs/vetur/blob/master/.github/NO_REPRO_CASE.md

**Comment by willhoyle** [comment]
thanks for looking into this. I hope the following repro helps:

https://github.com/willhoyle/veturpack

Directions are in the readme

**Comment by yoyo930021** [comment]
> thanks for looking into this. I hope the following repro helps:
> 
> https://github.com/willhoyle/veturpack
> 
> Directions are in the readme

Thank you for your help.
I fix it in v0.31.3.
